<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-OSSH23AN" failures="2" tests="4" name="PetTests" time="9.912" errors="0" timestamp="2025-06-04T12:31:37 IST">
  <testcase classname="api.test.UserTests2" name="testPostUser" time="3.157"/>
  <testcase classname="api.test.UserTests2" name="testGetUserByName" time="1.544">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testGetUserByName(UserTests2.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests2" name="testUpdateUserByName" time="2.892"/>
  <testcase classname="api.test.UserTests2" name="testDeleteUserByName" time="1.423">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testDeleteUserByName(UserTests2.java:106)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- PetTests -->
